### DISCUSSION TITLE
GeoSam interactive map can't find file or path when generating segments

### DISCUSSION THREAD

**Discussion by AdnanKhayyat1** [opengeos/segment-geospatial#267] [Q&A]
Whenever I set the pins and click to generate segments I keep getting the error `invalid path or file: None`

Here's the notebook cells for the interactive map and model: 

```
sam = SamGeo(
    model_type="vit_h",
    automatic=False,
    sam_kwargs=None,
    checkpoint="model/sam-h-model.pth",
    output_dir="/images/",
)
```

```
m = sam.show_map(out_dir="/images/masks.tif")
m.add_basemap("SATELLITE")
m
```
<img width="613" alt="Screenshot 2024-04-16 at 3 36 20 PM" src="https://github.com/opengeos/segment-geospatial/assets/37910611/88756690-060e-4854-8840-50c89ed7b2af">

**Comment by giswqs** [comment]
Try removing the out_dir param for the show_map

**Reply by AdnanKhayyat1** [reply]
Still doesn't work unfortunately. Putting `out_dir` was an attempt to fix this.

**Comment by giswqs** [comment]
Did you try it using Colab?
https://colab.research.google.com/github/opengeos/segment-geospatial/blob/main/docs/examples/input_prompts.ipynb
